src/client/components/BackButton/BackButton.tsx

```diff
@@ -16,15 +16,16 @@ const Button = styled(UnstyledButton)`
   }
 `
 
-export const BackButton = () => {
+export const BackButton = ({ onClick }: { onClick: () => void }) => {
```

Comment:
Should perhaps be moved to a `BackButtonProps`? 

Perhaps this button is only ever used in `CheckoutPageWrapper`? If not, should `onClick` be optional?

Reply:
Ok, will do that! :D I will make this PR into a draft again now and await Marian's hook to be merged, so there will be some more changes to this soon!